sig-robotics/proposal/Generating Action Sequences for Robot Task Plans using LLM/README.md

```diff
@@ -0,0 +1,176 @@
+<center>
+    <h1>Generating Action Sequences for Robot Task Plans using LLM</h1>
+</center>
+
+
+# 1. Background
+Since Google proposed the SayCan framework, empowering robots with large language models to interpret complex task instructions has become a popular research topic. The demand for robots is no longer limited to simple command execution; instead, there is a desire for semantic understanding and task execution based on the exploration of underlying needs. Technologies like ChatGPT have driven the advancement of this field, while RoboSDK and ROS, which provide a unified API for cloud-based, robot-centric applications, can fulfill data collection and task execution requirements for heterogeneous devices.
+
+This project use **prompt technique** to drive complex Robot operations, user utilize a prompt like  `Go to the kitchen and help me get an apple.` This prompt makes Robot go to the kitchen and find the apple, take it away, and finally take it back to the owner.
+
+Current research：
+- google [SayCan](https://say-can.github.io/)
+- Microsoft [PromptCraft-Robotics](https://github.com/microsoft/PromptCraft-Robotics/tree/main)
+- Nvidia [VOYAGER](https://github.com/MineDojo/Voyager)
+
+This proposal provides a basic robot LLM Agent (RoboAgent) based on [RoboSDK](https://github.com/KubeEdge4Robotics/robosdk), enabling robot developers and users to use LLM prompt to drive robots in an intelligent manner.
+
+# 2. Goals
+For developers or end users of RoboSDK, the goals of the RoboAgent are:
+
+1. Test the perception, planning, and control algorithms of a robot in complex navigation and grasping tasks, both indoors and outdoors, based on given instructions (e.g. "Go to the kitchen and help me get an apple."). Utilize large language models like gpt-3.5-turbo for this purpose.
+
+2. Perform closed-loop verification of the developed algorithms in **[Gazebo](https://gazebosim.org/home)** simulation environment.
+
+3. Deploy the data transmission model and other development processes using RoboSDK.
```

Comment:
This is what function this proposal would use, not the goal it try to achieve. That means using RoboSDK is just a choice. So I suggest to descript what function it provided, not what function it used. This line can be descriped like “Provides data collection for something”, or at least "Integrates data collection funciton by something"